Sources/Attribution/AttributionPoster.swift

```diff
@@ -154,6 +154,10 @@ final class AttributionPoster {
         Self.postponedAttributionData = nil
     }
 
+    func clearPostponedAttributionData() {
+        Self.postponedAttributionData = nil
+    }
```

Comment:
This is not strictly required, but I don't think it harms to clear the postponed attribution data when the SDK is initialized in UI preview mode.

Reply:
You are right. Yes, I also think it's fine because UI preview mode is internal. But I'm not really sure about needing to have the `clearPostponedAttributionData()` method anyway, as in UI preview mode the `postponedAttributionData` should never populate with data 🤔